plotsquared/api/api-documentation.md

```diff
@@ -0,0 +1,161 @@
+# API Documentation
+
+## Maven & Gradle Examples
+
+* Javadocs: [here](https://intellectualsites.github.io/plotsquared-javadocs)
+* Major upgrade diff: [here](https://intellectualsites.github.io/plotsquared-api-diff)
```

Comment:
Don't use "here" as placeholder for URLs => https://uxmovement.com/content/why-your-links-should-never-say-click-here/